<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OverviewPage</class>
 <widget class="QWidget" name="OverviewPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="cwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>696</width>
     <height>460</height>
    </rect>
   </property>
   <widget class="QWidget" name="infowidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>7</y>
      <width>503</width>
      <height>206</height>
     </rect>
    </property>
    <widget class="QLabel" name="lb_news">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>6</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>最新动态：</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="tb_webinfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>481</width>
       <height>161</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;最新资讯信息获取中...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_amount">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>281</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>金额</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_opendir">
    <property name="geometry">
     <rect>
      <x>539</x>
      <y>155</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;打开您的区块链文件夹&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>日期</string>
    </property>
   </widget>
   <widget class="QPushButton" name="unlockWalletButton">
    <property name="geometry">
     <rect>
      <x>539</x>
      <y>86</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;锁定/解锁钱包&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="ov_topwidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>231</y>
      <width>678</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QLabel" name="ov_label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>20</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ov_label_10">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>最近交易记录</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTransactionsStatus">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>(数据同步中)</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelNumTransactions">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ov_label_11">
     <property name="geometry">
      <rect>
       <x>513</x>
       <y>10</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>钱包内交易数：</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_addr">
    <property name="geometry">
     <rect>
      <x>309</x>
      <y>280</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>地址</string>
    </property>
   </widget>
   <widget class="QListView" name="listTransactions">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>311</y>
      <width>678</width>
      <height>140</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_setpwd">
    <property name="geometry">
     <rect>
      <x>539</x>
      <y>20</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;设置/修改钱包密码&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
